src/elm/Pages/Pipeline/View.elm

```diff
@@ -221,35 +236,58 @@ wrapPipelineConfigurationContent model cls content =
         body
 
 
-{-| viewTemplatesExpansion : takes model and renders the config header button for expanding pipeline templates.
+velaYmlFileName =
+    "vela.yml"
```

Comment:
funny thing, I wanted to download the file as a hidden dot vela.yml `.vela.yml` but the browser will just ignore the `.` if its at the beginning of a filename anyways!!!